toplevel/coqloop.ml

```diff
@@ -292,6 +278,9 @@ let coqloop_feed (fb : Feedback.feedback) = let open Feedback in
   | FileDependency (_,_) -> ()
   | FileLoaded (_,_) -> ()
   | Custom (_,_,_) -> ()
+  (* Re-enable when we switch back to feedback-based error printing *)
```

Comment:
Why is that fishy? We cannot rely on feedback ATM, for example:
```
Goal True.
Goal u.
```
won't produce a feedback error as noted by @herbelin . cf 5479